### PR TITLE
Allow the Action Manager to acces Cloudbase-Init Config

### PR DESCRIPTION
**Pull request by mateimicu**
Signed-off-by: Micu Matei-Marius mmicu@cloudbasesolutions.com

### CODE REVIEW

File: argus/action_manager/base.py [hunk 1]
```diff
@@ -147,3 +147,16 @@ def touch(self, path):
         as side effect.
```
Comment (ader1990) [inline argus/action_manager/base.py:151]: don't put in the commit message references to yourself

**Review by ader1990 — CHANGES_REQUESTED** [review]
Shouldn't we create a specific action manager for windows nano?

File: argus/action_manager/windows.py [hunk 2]
```diff
@@ -339,6 +339,17 @@ def touch(self, path):
         else:
             self.mkfile(path)
 
+    # pylint: disable=unused-argument
```
Comment (stefan-caraiman) [inline argus/action_manager/windows.py:367]: Is this pragma still required?

File: argus/action_manager/base.py (hunk 1 above)
Comment (mateimicu) [reply argus/action_manager/base.py:151]: Done.

File: argus/action_manager/windows.py (hunk 2 above)
Comment (mateimicu) [reply argus/action_manager/windows.py:367]: Still not using `cbinit_conf` or `cbinit_unattend_conf` in this method.

**Comment by alexcoman** [comment]
LGTM;

**Review by alexcoman — APPROVED** [review]
LGTM

**Pull request merged**